src/Unleash.php

```diff
@@ -2,8 +2,7 @@
 
 namespace MikeFrancis\LaravelUnleash;
 
-use function GuzzleHttp\json_decode;
```

Comment:
Is there a reason this was removed? I was using it as I thought it was quite handy that it throws an exception if the JSON is invalid

Reply:
Ah, I just noticed it was deprecated and removed it, I added it back while removing the `\RuntimeException` below